src/_colors.scss

```diff
@@ -1,6 +1,7 @@
 /* UI Primary Colors */
 $ui-blue: hsla(215, 100, 65, 1); // #4C97FF Motion Primary
 $ui-blue-dark: hsla(215, 65, 55, 1); // #3373CC Motion Secondary
+$ui-blue-very-light:  hsla(215, 100, 95, 1);
```

Comment:
@benjiwheeler - Generally we try not to add color variables, we already have an excess. But that's on me, I should have called out that the background of the container should be a transparent blue: `$ui-blue-10percent`

Reply:
The problem is the transparency -- behind it is purple! But I'll see if I can replace that with white.